website/docs/docs/introduction.md

```diff
@@ -3,188 +3,65 @@ title: "What is dbt?"
-
-These data sources, known as seed files, can be saved as a CSV file in your `project` and loaded into your data warehouse through use of the `seed` command.
-
-The documentation for the seed command can be found [here](seeds).
-
-
-**Data snapshots:**
-Often, records in a data source are mutable, in that they change over time. This can be difficult to handle in analytics if you want to reconstruct historic values.
-
-dbt provides a mechanism to snapshot raw data for a point in time, through use of [snapshots](snapshots).
-
-
-**Understand raw data sources:**
-Since dbt selects from raw data already loaded in your data warehouse, it's useful to understand the role of these tables and schemas within your warehouse. Defining raw data as a Source in dbt allows you to:
-* Document and test the data that is loaded into your warehouse
-* Check the freshness of your data against specified limits, to help identify upstream issues.
-* Understand which models depend on each data source.
-
-Check out the [documentation on Sources](using-sources).
-
-
-## Who should use dbt?
-dbt is appropriate for anyone who interacts with a data warehouse. It can be used by data engineers, data analysts and data scientists, or anyone that knows how to write select queries in SQL.
-
-For dbt users that are new to programming, you may also need to spend some time getting to know the basics of the command line, and familiarizing yourself with git.
-
-To make full use of dbt, it may also be beneficial to know some programming basics, such as `for` loops and `if` statements, to use Jinja effectively in your models.
+| Feature               | Description |
+|-----------------------|-------------|
+| Handle boilerplate code to materialize queries as relations | For each model you create, you can easily configure a *materialization*. A materialization represents a build strategy for your select query – the code behind a materialization is robust, boilerplate SQL that wraps your select query in a statement to create a new, or update an existing, relation. Read more about [Materializations](/docs/build/materializations).|
+| Use a code compiler | SQL files can contain Jinja, a lightweight templating language. Using Jinja in SQL provides a way to use control structures in your queries. For example, `if` statements and `for` loops. It also enables repeated SQL to be shared through `macros`. Read more about [Macros](/docs/build/jinja-macros).|
+| Determine the order of model execution | Often when transforming data, it makes sense to do so in a staged approach. dbt provides a mechanism to implement transformations in stages through the [ref function](/reference/dbt-jinja-functions/ref). Rather than selecting from existing tables and views in your warehouse, you can select from another model.|
```

Comment:
```suggestion
| Determine the order of model execution | Often, when transforming data, it makes sense to do so in a staged approach. dbt provides a mechanism to implement transformations in stages through the [ref function](/reference/dbt-jinja-functions/ref). Rather than selecting from existing tables and views in your warehouse, you can select from another model.|
```